consumer/src/test/kotlin/streams/StreamsEventSinkQueryExecutionTest.kt

```diff
@@ -0,0 +1,43 @@
+package streams
+
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.neo4j.graphdb.GraphDatabaseService
+import org.neo4j.kernel.internal.GraphDatabaseAPI
+import org.neo4j.test.TestGraphDatabaseFactory
+import streams.kafka.KafkaSinkConfiguration
+import kotlin.test.assertEquals
+
+class StreamsEventSinkQueryExecutionTest {
+    private lateinit var db: GraphDatabaseService
+    private lateinit var streamsEventSinkQueryExecution: StreamsEventSinkQueryExecution
+
+    @Before
+    fun setUp() {
+        db = TestGraphDatabaseFactory()
+                .newImpermanentDatabaseBuilder()
+                .newGraphDatabase()
+        val kafkaConfig = KafkaSinkConfiguration(streamsSinkConfiguration = StreamsSinkConfiguration(topics = mapOf("shouldWriteCypherQuery" to "MERGE (n:Label {id: event.id})\n" +
```

Comment:
do we also have a plain StreamsSinkConfiguration test?

Reply:
Done